docs/hardware.rst

```diff
@@ -272,39 +272,13 @@ Specific Hardware Recommendations
 Application and Monitor Servers
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We currently recommend the Gigabyte BRIX, Intel NUC, or Mac Mini for
-SecureDrop servers.
+We currently recommend the Intel NUC for SecureDrop servers.
```

Comment:
Leaving out the Mac Mini altogether here is probably safest. it's not like the 2014s will be easily commercially available much longer.